query/read/fields.go

```diff
@@ -37,6 +38,37 @@ func BuildFields(reqFields jsoniter.RawMessage) (*FieldFactory, error) {
 	factory.Include = make(map[string]Field)
 	factory.Exclude = make(map[string]Field)
 
+	var v interface{}
+	if err := json.Unmarshal(reqFields, &v); err != nil {
+		return nil, err
+	}
+
+	var err error
+	switch v := v.(type) {
```

Comment:
What will we have in the open API spec? Object or array?

Reply:
if it is confusing, I am fine to stick with having it as an object to avoid any confusion and if needed in clients we can have an array.